app/src/main/java/org/mozilla/fenix/ext/Context.kt

```diff
@@ -64,6 +64,9 @@ fun Context.asFragmentActivity() = (this as? ContextThemeWrapper)?.baseContext a
 fun Context.getPreferenceKey(@StringRes resourceId: Int): String =
     resources.getString(resourceId)
 
+@Suppress("unused")
+fun Context.isInExperiment(@Suppress("UNUSED_PARAMETER") experimentName: String): Boolean = false
```

Comment:
Nit: would you mind adding a comment along the lines of `Feature flags are not supported by Experimentor, we are currently looking into another way to handle them. See {issue number}`